bigchaindb/tendermint/core.py

```diff
@@ -19,6 +20,29 @@ def __init__(self, bigchaindb=None):
         if not bigchaindb:
             bigchaindb = BigchainDB()
         self.bigchaindb = bigchaindb
+        self.block_txn_ids = []
+        self.block_txn_hash = ''
+        self.validators = None
+        self.new_height = None
+
+    def init_chain(self, validators):
+        """Initialize chain with block of height 0"""
+        print("\ninit_chain::\n")
```

Comment:
I'd use `log`. Or you can even remove the `print` for now, and we address logging later.

Reply:
Sure, will remove it. Was using it for testing locally.